src/main/java/hudson/plugins/cobertura/MavenCoberturaBuildAction.java

```diff
@@ -1,32 +0,0 @@
-package hudson.plugins.cobertura;
-
-import hudson.maven.AggregatableAction;
-import hudson.maven.MavenAggregatedReport;
-import hudson.maven.MavenBuild;
-import hudson.maven.MavenModule;
-import hudson.maven.MavenModuleSetBuild;
-import hudson.plugins.cobertura.targets.CoverageResult;
-import hudson.plugins.cobertura.targets.CoverageTarget;
-
-import java.util.List;
-import java.util.Map;
-import java.util.logging.Logger;
-
-/**
- * Created by IntelliJ IDEA.
- *
- * @author connollys
- * @since 18-Dec-2007 11:11:23
- */
-public class MavenCoberturaBuildAction extends CoberturaBuildAction implements AggregatableAction {
```

Comment:
The plot thickens. Even before #130, acc. to https://github.com/jenkinsci/cobertura-plugin/blob/c2a5b1d30ed2d7572f94a59110a7fcfe905f9c6f/src/main/java/hudson/plugins/cobertura/MavenCoberturaPublisher.java#L267-L268 + https://github.com/jenkinsci/cobertura-plugin/blob/c2a5b1d30ed2d7572f94a59110a7fcfe905f9c6f/src/main/java/hudson/plugins/cobertura/CoberturaBuildAction.java#L276 this class, and thus also `MavenCoberturaAggregatedReport`, was not used either.

Reply:
And AFAICT this has been true since bd721c6b83a2d2a47a71bffc0d6fb05d46ff99ff. :man_shrugging: this is what happens when you have no test coverage—ironically enough for this plugin!